api/scripts/ariImport.ts

```diff
@@ -1,39 +1,64 @@
 import axios from 'axios';
-import * as fs from 'fs/promises';
 import * as dotenv from 'dotenv';
-import { expand } from 'dotenv-expand';
 
 // Important to do this so that environment variables are treated the same as in deployed code.
-expand(dotenv.config());
```

Comment:
This was due to serverless right? Guessing this won't be running there anymore?

Reply:
Yeah, from a bit of experimenting it doesn't seem to anymore. The default behaviour now is as if using dotenv, but not dotenv-expand.